spec/javascripts/health-care/utils/validations.spec.js

```diff
@@ -59,4 +59,17 @@ describe('Validations unit tests', () => {
       expect(isValidDate(0, 2, 2016)).to.be.false;
     });
   });
+
+  describe('isValidMonetaryValue', () => {
+    it('validates monetary values', () => {
+      expect(isValidMonetaryValue('100')).to.be.true;
+      expect(isValidMonetaryValue('1.99')).to.be.true;
+      expect(isValidMonetaryValue('1000')).to.be.true;
+      expect(isValidMonetaryValue('')).to.be.true;
```

Comment:
I was thinking yes because not all monetary value fields are required, so we want to return false only if they enter something invalid.